src/Uno.UI.Tests/Windows_UI_XAML_Controls/GridTests/Given_Grid_And_Min_Max.cs

```diff
@@ -0,0 +1,344 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.UI.Tests.Windows_UI_XAML_Controls.GridTests
+{
+	[TestClass]
+	public class Given_Grid_And_Min_Max
+	{
+		[TestMethod]
+		public void When_Max_Width_Star()
```

Comment:
All those tests look like good candidates for:

https://github.com/nventive/Uno/blob/8dcc6f2d3aa5d57fd3ee96efe5fe73855a45a0da/src/Uno.UI.Tests/Resources/Given_ResourceCandidate.cs#L9

Not a big deal though :)

Reply:
Yeah true

Reply:
I update the related internal issue ;) Thanks @davidjohnoliver, nice work!